src/main/java/gov/cabinetoffice/gap/adminbackend/services/SchemeEditorService.java

```diff
@@ -111,4 +113,17 @@ public SchemeEntity addEditorToScheme(final Integer schemeId, final String edito
         return this.schemeRepo.save(scheme);
     }
 
+    public void deleteEditor(final Integer schemeId, final Integer editorId) {
+        final SchemeEntity scheme = schemeRepo.findById(schemeId)
+                .orElseThrow(() -> new NotFoundException("Delete scheme editor: Scheme not found"));
+
+        if (scheme.getCreatedBy().equals(editorId))
+            throw new ForbiddenException("Delete scheme editor: Cannot delete scheme creator");
+
+        final GrantAdmin grantAdmin = grantAdminRepository.findById(editorId)
+                .orElseThrow(() -> new NotFoundException("Delete scheme editor: Grant Admin with editor access not found for this scheme"));
+
+        scheme.removeAdmin(grantAdmin);
```

Comment:
What happens if the admin exists but is not an editor on the scheme?

Reply:
As per check above this - admin must own this scheme. Editors should not be able to remove other editors as per the ACs